### ISSUE TITLE
`Uri::parseQueryString()` seems broken `"a=a%20a" => ['a' => 'a%2520a']`

### ISSUE THREAD

**Issue by LastDragon-ru** [opis/uri#1]
Looks like it encodes values instead of decoding them :)

```
    public function testParseQueryString(): void {
        $params = ['a' => 'a a'];
        $query  = http_build_query($params, encoding_type: PHP_QUERY_RFC3986);
        $parsed = Uri::parseQueryString($query);

        $this->assertEquals($params, $parsed);
    }
```

![image](https://user-images.githubusercontent.com/1329824/119231596-bdae2e80-bb32-11eb-91e6-816f612f947b.png)

**Comment by sorinsarca** [comment]
Fixed on master branch, a new version is coming soon

**Comment by sorinsarca** [comment]
Fixed in v1.1.0
